### PR TITLE
Adds open auth announcement blog

### PR DESCRIPTION
**Pull request by kendallstrautman**
Todo: 
~~- [ ] Add consumer metadata~~ - since the packages aren't in Tina this isn't appropriate yet
- [x] Add any media assets - gif could be better. ~~thinking about redoing it when the toolbar is done~~
- [ ] Final edits by everyone

### CODE REVIEW

File: content/blog/tina-style-open-authoring.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+> If yew sea ani speling misteaks, pleeze ficks thim. Wee wood apprisheate that vary much 😂.
+
+## Using [Next.js](nextjs.org) to enable "Edit-mode"
+
+Our ability to execute this idea coincided with updates for [Next.js](https://nextjs.org/). The ZEIT team has been polishing a "preview" feature that was instrumental for our Open Authoring prototype. We're using it to switch a static site into "Edit Mode". It's quite magical — a testament to their high-quality engineering. In a few words, this is how it works:
+
+<img src="" alt="diagram of edit mode">
+
+Before this "preview" feature, page templates in Next.js were either static or used Server-Side Rendering (SSR). Now, the same page template can take advantage of **both static and SSR data-loading**. This allows us to switch “content” data-sources without rebuilding the site.
```
Comment (jamespohalloran) [inline content/blog/tina-style-open-authoring.md]:
```suggestion
Before this "preview" feature, page templates in Next.js were either static or used Server-Side Rendering (SSR). Now, the same page template can conditionally take advantage of **both static and SSR data-loading**. This allows us to switch “content” data-sources without rebuilding the site.
```

File: content/blog/tina-style-open-authoring.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Any time an editor loads a page in "Edit Mode", we can build a unique page for them. This page get its own data from a particular Git branch or fork. SSR renders the page and enables Tina editing magic 🧙‍♀️. This is how we can make content changes right on the page, with a familiar Git-based workflow under the hood.
```
Comment (jamespohalloran) [inline content/blog/tina-style-open-authoring.md]:
```suggestion
Any time an editor loads a page in "Edit Mode", we can build a unique page for them. This page gets its own data from a particular Git branch or fork. SSR renders the page with the user's dynamic content, so we can even allow the user to create new pages that don't exist in the static build.
```
I'm not super high on these edits, but the original makes it sound like SSR is required for Tina to work. It's really only required for dynamic data (unless we want to refetch on the client), and for rendering new pages that don't exist statically.

File: content/blog/tina-style-open-authoring.md (hunk 1 above)
Comment (jamespohalloran) [reply content/blog/tina-style-open-authoring.md]: May be a risk for information overload, but we could emphasize that supporting both static and SSR means that we don't need a separate "cloud editing environment" anymore

File: content/blog/tina-style-open-authoring.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+We're focused on improving the independent web as a whole. We want to craft tools that help people build better sites and create better content.
+
+In-line with this mission, we built **Tina-Style Open Authoring** ✍️. [Netlify CMS](https://www.netlifycms.org/) spearheaded the [core idea](https://css-tricks.com/netlify-cms-open-authoring/#article-header-id-0) this past summer. It involves opening up your website to accept content-related _contributions from anyone_.
+
+## Picture this:
+
+On your website, there's an _"Edit this page on GitHub"_ link — familiar enough. But imagine that when you click it, instead of kicking you over to GitHub, the site itself becomes editable — **like a Google Doc**.
+
+The scaffolding underneath Open Authoring is a Git-based workflow 😍. Triggering 'Edit Mode' creates a new fork to track and commit changes on. When content updates are ready, the editor can make a pull-request. The site owner can then review the work before making it live.
```
Comment (jamespohalloran) [inline content/blog/tina-style-open-authoring.md]:
```suggestion
The scaffolding underneath Open Authoring is a Git-based workflow. Triggering 'Edit Mode' creates a new fork to track and commit changes on. When content updates are ready, the editor can make a pull-request. The site owner can then review the work before making it live.
```
Maybe Nitpicky, but at this point, I might not understand why this is heart-worthy until I read more

Comment (kendallstrautman) [reply content/blog/tina-style-open-authoring.md]: lolz

File: content/blog/introducing-visual-open-authoring.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+title: Introducing Visual Open Authoring
+date: '2020-03-05T07:00:00.000Z'
```
Comment (kendallstrautman) [inline content/blog/introducing-visual-open-authoring.md]:
```suggestion
date: '2020-03-09T07:00:00.000Z'
```

**Pull request merged**